core/src/main/java/org/mule/processor/chain/AbstractMessageProcessorChain.java

```diff
@@ -230,4 +232,16 @@ public void setMuleContext(MuleContext context)
             }
         }
     }
+
+    @Override
+    public boolean isNonBlocking(MuleEvent event)
+    {
+        return false;
```

Comment:
Do this means that adding any message processor chain will convert the processing to blocking?